app/api/files/jsRoutes.js

```diff
@@ -76,6 +85,11 @@ export default app => {
           socket(req).emit('documentProcessed', newEntity.sharedId);
         });
       }
+
+      if (req.body.mail) {
+        mailer.send({ from: `"${req.body.mail.sender}" <no-reply@uwazi.io>`, ...req.body.mail });
```

Comment:
The validation defines the "email" key object, yet here you use the "mail" key.  How is this even working?

Reply:
it wasn't